understanding/21/non-text-contrast.html

```diff
@@ -192,22 +186,22 @@ <h4>User Interface Component Examples</h4>
 
 					<table class="left-headers">
 						<caption>
-							User Interface Component Examples
+							Passing User Interface Component Examples
 						</caption>
 						<tr>
 						<th>Type</th>
 						<th>Description</th>
 						<th>Examples</th>
 						</tr>
 						<tr>
-							<th>Link Text</th>
+							<th>Default link text</th>
 							<td>Default link text is in the scope of <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>, and the underline is sufficient to indicate the link.</td>
```

Comment:
@fstrr This addresses the concerns we discussed in an email conversation on June 20
```suggestion
							<td>Default link text is covered by <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>,  not Non-text Contrast. Regardless of whether one considers the underline part of the text or a non-text indicator, since it is the same color as the text, which must meet 4.5:1 to pass, the default underline will always pass the lower contrast requirements of Non-text Contrast.</td>
```

Reply:
Thanks, Mike.

What about this, which is a little more succinct?:

```suggestion
							<td>The browser's default link text color is covered by <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>. Since the underline is the same color as the text, which must meet at least 3:1 to pass, the default underline will always pass the requirements of Non-text Contrast.</td>
```